shardingsphere-kernel/shardingsphere-transaction/shardingsphere-transaction-core/src/main/java/org/apache/shardingsphere/transaction/ConnectionTransaction.java

```diff
@@ -46,6 +48,7 @@ public ConnectionTransaction(final String databaseName, final TransactionRule ru
     }
     
     public ConnectionTransaction(final String databaseName, final TransactionType transactionType, final TransactionRule rule) {
+        this.databaseName = databaseName;
```

Comment:
Should databaseName be extracted from SQL?

Reply:
I will check.